### ISSUE TITLE
Can't see 'remove attachment' button over some media

### ISSUE THREAD

**Issue by TodePond** [elk-zone/elk#755]
I can't see the 'remove attachment' button on some attachments:

![image](https://user-images.githubusercontent.com/15892272/210530829-ae35b085-7880-4d0d-abac-203d741636ff.png)

**Comment by boehs** [comment]
It's now 2 colours, so should be fine

![image](https://user-images.githubusercontent.com/51836263/216790181-accced00-cc68-4240-87d3-77533c6df350.png)
